docs/app/Component/ComponentList.js

```diff
@@ -17,7 +17,7 @@ export default class ComponentList extends Component {
       .map(name => {
         let doc = new ComponentDoc(name);
         return (
-          <Segment key={doc.name} id={doc.name} className='vertical'>
+          <Segment key={doc.name} id={doc.name}>
```

Comment:
Removed the vertical class so each component segment is the default style.  Gives depth separation between component doc sections:

![image](https://cloud.githubusercontent.com/assets/5067638/10407894/332dae22-6ea6-11e5-83cb-29dda77bd93b.png)